keps/sig-node/2400-node-swap/README.md

```diff
@@ -921,15 +926,22 @@ Here are specific improvements to be made:
 - Add e2e test confirming that swap is used for `LimitedSwap`.
 - Document [best practices](#best-practices) for setting up Kubernetes with swap
 
-[via cgroups]: #restrict-swap-usage-at-the-cgroup-level
+#### Beta 3
 
-#### GA
+- Enhance website documentation
+  - Docs should be close to GA quality before promoting to GA.
+- Make eviction manager swap aware.
+- Test a wide variety of scenarios that may be affected by swap support, including tests with aggressive memory stress.
+- Address memory-backed backed volumes which should not have access to swap.  
+- Add documentation regarding encrypted swap.
+- Test a wide variety of scenarios in which the node is being memory-stressed with different eviction and swap configurations. 
+Ensure the behavior is consistent and reliable.
+- Exclude high-priority, static and mirrored pods from gaining access to swap.
 
-_(Tentative.)_
+#### GA
 
-- Test a wide variety of scenarios that may be affected by swap support.
-- Remove feature flag.
-- Remove the Swap Support using Burstable QoS Pods only deprecated in Beta 2.
+- Remove feature gate.
+- Ensure swap-conformance tests are continuously green.
```

Comment:
@SergeyKanzhelev @dchen1107

Would you be able to kindly review this section specifically? Do you see anything missing with these graduation requirements?

As this KEP update suggests, my aim is to graduate to beta3 in 1.33 and GA in 1.34.

Reply:
FYI @yujuhong